.circleci/run_terraform.sh

```diff
@@ -65,9 +65,14 @@ elif [[ $branch == "dev" ]]; then
     ENVIRONMENT=staging
 else
     echo "Why in the world was run_terraform.sh called from a branch other than dev or master?!?!?"
-    exit 1
+    # TODO: uncomment this
+    # exit 1
```

Comment:
??

Reply:
These are just like this until we've tested this because if this isn't here then test deploys fail. (You can't just comment it out locally for testing since the actual branch is checked out onto the deploy box as part of the deploy process.)